src/masternode-sync.cpp

```diff
@@ -270,7 +192,19 @@ void CMasternodeSync::Process()
 
     if (tick++ % MASTERNODE_SYNC_TIMEOUT != 0) return;
 
-    if (IsSynced()) {
+    // if the last call to this function was more than 60 minutes ago (client was in sleep mode)
+    // reset the sync process
+    int64_t now = GetTime();
+    if (lastProcess != 0 && now > lastProcess + 60 * 60) {
+        Reset();
+        g_tiertwo_sync_state.SetBlockchainSync(false, 0);
```

Comment:
`Reset()` already sets the blockchain-sync to false, no need to do it again here (not introduced in this PR, but it's a good opportunity to fix it).

Reply:
sure